### ISSUE TITLE
Tapping on Upvote or Downvote with active transaction close "Waiting for confirmation" banner

### ISSUE THREAD

**Issue by churik** [dap-ps/discover#15]
### Reproduction

- Open dev.dap.ps
- Tap "Upvote" on any dapp, sign transaction
- Tap on Upvote or Downvote one more time

#### Expected behavior
banner with transaction is still shown

#### Actual behavior
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/4557972/62774567-5fbb6e80-baa5-11e9-99bb-c3628278f5c8.gif)

### Additional Information

[comment]: # (Please do your best to fill this out.)
- Status version: nightly 09/08/2019

- Operating System: Android, iOS

**Comment by andytudhope** [comment]
Yeah, I think we can make the transaction queue and that tracking box more clever in general. @Perseverance - you had some ideas on this. Would you like me to make a bounty here for this iisue specifically for you and Limechain to take on at your leisure?

**Comment by iamonuwa** [comment]
The confirmation is called before the try and catch block.
